lms/djangoapps/teams/static/teams/templates/team-member.underscore

```diff
@@ -0,0 +1,8 @@
+
+<span class="team-member">
+    <a class="member-profile" href="<%= memberProfileUrl %>">
+        <p><%- gettext(username) %></p>
+        <img class="image-url" src="<%= imageUrl %>" alt="<%- gettext(username) %>" />
```

Comment:
The alt text here is redundant since the exact same text precedes it.  I would make the alt text here, the translatable string "profile page" so the screen reader reads  "@muzaffaryousaf profile page".  Currently, it would read "muzaffaryousaf muzaffaryousaf"